bin/nim-gdb

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env bash
-
-# Exit if anything fails
-set -e
-
-which nim > /dev/null || (echo "nim not in PATH"; exit 1)
-which gdb > /dev/null || (echo "gdb not in PATH"; exit 1)
-which readlink > /dev/null || (echo "readlink not in PATH."; exit 1)
-
-if [[ "$(uname -s)" == "Darwin" || "(uname -s)" == *"BSD" ]]; then
-  NIM_SYSROOT=$(dirname $(dirname $(readlink -f $(which nim))))
-else
-  NIM_SYSROOT=$(dirname $(dirname $(readlink -e $(which nim))))
-fi
-
-# Find out where the pretty printer Python module is
-GDB_PYTHON_MODULE_PATH="$NIM_SYSROOT/tools/nim-gdb.py"
-
-# Run GDB with the additional arguments that load the pretty printers
-# Set the environment variable `NIM_GDB` to overwrite the call to a
-# different/specific command (defaults to `gdb`).
-NIM_GDB="${NIM_GDB:-gdb}"
-# exec replaces the new process of bash with gdb. It is always good to
-# have fewer processes.
-exec "${NIM_GDB}" -eval-command="source $GDB_PYTHON_MODULE_PATH" "$@"
```

Comment:
Why is this file marked to be removed?

Reply:
this was done pretty rushed-ly - i commited whatever was in my staging area, in case my pc caught fire.
 don't worry about it too much haha

Reply:
Well I do worry about it. This I wrote this file and it was attacked in the past from left and right. Hand to restore its functionality multiple times. Couldn't find a way to automatically test for this file to work in CI.

Reply:
Understandable. In its final form, this pr will restore your file to its former glory